### PR TITLE
CASMTRIAGE-2681: Changed hardware name from Server to Node for Gigabyte and HPE

### PR DESCRIPTION
**Pull request by haroldlongley**
## Summary and Scope

CASMTRIAGE-2681: Changed hardware name from Server to Node for Gigabyte and HPE

Software in the system has client/server relationship.  But calling a compute node a compute "server" causes confusion about the role of the component.

A server can be standalone.  A node is part of a collection of components such as a supercomputer.

### CODE REVIEW

File: operations/node_management/Update_the_Gigabyte_Node_BIOS_Time.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+
+## Update the Gigabyte Node BIOS Time
```
Comment (nrockershousen) [inline operations/node_management/Update_the_Gigabyte_Node_BIOS_Time.md:2]: An entry for this file should be added to the Node Management section in the /operations/index.md file.

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
Friendly housekeeping note! This needs to be rebased/cherry-picked before it can be reviewed. Or (given the age of this PR), if the change is no longer needed or applicable, please close this badboy.

**Comment by rustydb** [comment]
Closing this PR, it is stale and the JIRA is either incorrect or has been moved (I Can't find CASMTRIAGE-2681).

**Pull request closed without merging**